internal/storage/activities/aip_deletion_report.go

```
@@ -51,38 +60,35 @@ func (a *AIPDeletionReportActivity) Execute(
 	if a.cfg.ReportTemplatePath == "" {
 		return nil, fmt.Errorf("AIP deletion report: template path is not configured")
 	}
+	if _, err := os.Stat(a.cfg.ReportTemplatePath); err != nil {
+		if os.IsNotExist(err) {
+			return nil, fmt.Errorf("AIP deletion report: template file does not exist: %s", a.cfg.ReportTemplatePath)
+		}
+		return nil, fmt.Errorf("AIP deletion report: error reading template file: %s", err)
```

I'd just return the last error in both cases, not a big difference.

---

Sure.

---

Using the last error message in both cases changes: 
"AIP deletion report: template file does not exist: non_existent.tmpl.pdf", 

to: 
"AIP deletion report: error reading template file: stat non_existent.tmpl.pdf: no such file or directory"

Which is definitely a bit harder to read.  I think I'll stick with the two separate messages, though I will shorten down the second one.